.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```

Comment:
Are the JetBrains IDE files important to commit to the project? Shouldn't the entire .idea directory be excluded in the gitignore?

Reply:
Sometimes part of the .idea folder is committed so developers can share workspace settings, I can ignore and remove these files if you prefer.

Reply:
If there really is something of value in these project files, then maybe. But it feels weird committing files that make assumptions about IDEs. If we did that, then we might see projects with eclipse, sublime, vim, jetbrains, etc project files, accumulated over time from merge requests from different developers with different preferences.